metale_amorficzne/description/01_general.md

```diff
@@ -1,7 +1,272 @@
+attractive option for Armour material and other applications which requires high
+stress tolerance.[43][44][45] Additive manufacturing One challenge when
+synthesising a metallic glass is that the techniques often only produce very
+small samples, due to the need for high cooling rates. 3D-printing methods have
+been suggested as a method to create larger bulk samples. Selective laser
+melting (SLM) is one example of an additive manufacturing method that has been
+used to make iron based metallic glasses.[46][47] Laser foil printing (LFP) is
+another method where foils of the amorphous metals are stacked and welded
+together, layer by layer.
+
+---
+
+### What is nanoindentation?
+
+In an indentation test, a hard tip whose mechanical properties are known
+(frequently made of a very hard material like diamond) is pressed into a sample
+whose properties are unknown. The load placed on the indenter tip is increased
+as the tip penetrates further into the specimen and soon reaches a user-defined
+value. At this point, the load may be held constant for a period or removed. The
+area of the residual indentation in the sample is measured and the hardness, H
+{\displaystyle H}, is defined as the maximum load, P m a x {\displaystyle
+P_{max}}, divided by the residual indentation area, A r {\displaystyle A_{r}}:
+
+Since the mid-1970s nanoindentation has become the primary method for measuring
+and testing very small volumes of mechanical properties. Nanoindentation, also
+called depth sensing indentation or instrumented indentation, gained popularity
+with the development of machines that could record small load and displacement
+with high accuracy and precision. The load displacement data can be used
+to determine modulus of elasticity, hardness, yield strength, fracture
+toughness, scratch hardness and wear properties.
+
+In nanoindentation small loads and tip sizes are used, so the indentation area
+may only be a few square micrometres or even nanometres. This presents problems
+in determining the hardness, as the contact area is not easily found. Atomic
+force microscopy or scanning electron microscopy techniques may be utilized to
+image the indentation, but can be quite cumbersome. Instead, an indenter with a
+geometry known to high precision (usually a Berkovich tip, which has a
+three-sided pyramid geometry) is employed. During the course of the instrumented
+indentation process, a record of the depth of penetration is made, and then the
+area of the indent is determined using the known geometry of the indentation
+tip. While indenting, various parameters such as load and depth of penetration
+can be measured. A record of these values can be plotted on a graph to create a
+load-displacement curve (such as the one shown in Figure 1). These curves can be
+used to extract mechanical properties of the material.[3]
+
+---
+
+#### Materials and research methods used for this study
+
+- ZrCu and ZrTi alloys
```

Comment:
Add explanation what ZrCu and ZrTi is (i.e., expand the element names in parentheses).

Reply:
Done